community/index/src/main/java/org/neo4j/index/internal/gbptree/GBPTree.java

```diff
@@ -830,6 +842,32 @@ public void prepareForRecovery() throws IOException
         pagedFile.flushAndForce();
     }
 
+    /**
+     * Must be called after all recovered updates have been applied and before doing further work,
+     * be it reading or writing.
+     *
+     * @throws IOException on {@link PageCache} error.
+     */
+    public void completeRecovery() throws IOException
```

Comment:
this method name is a bit misleading since it's invoked from 'maybeCompleteRecovery' that invoked on start or during flush. So it's not always have something to do with recovery. What if we will call it clean for example?